src/main/webapp/script.js

```diff
@@ -19,41 +19,47 @@ function getRecipeInfo() {
   params.append('image', image);
   const request = new Request('/dishAnalysis', {method: "POST", body: params});
   fetch(request).then(response => response.json()).then((recipeListInfoJson) => {
-    sessionStorage.recipeList = JSON.parse(recipeListInfoJson);
-    window.location.href = "/display.html";
+    //set top options to radio buttons
+    document.getElementById("first-option-label").innerText = recipeListInfoJson[0];
+    document.getElementById("second-option-label").innerText = recipeListInfoJson[1];
+    //set radio button values
+    document.getElementById("first-option").value = recipeListInfoJson[0];
+    document.getElementById("second-option").value = recipeListInfoJson[1];
   });
 }
 
 /** Fetches and then populates nutrition information section of display page with average fat, calories, etc. */
 function createNutritionElements() {
+  dishName = sessionStorage.dishName;
   fetch('/dishNutrition?dishName='+dishName).then(response => response.json()).then((dish) => {
-    // Get dish name
-    var dishName = document.forms.dishFitChoice.elements.labelFitChoice.value;
-    if (dishName != null) {
-      title.setAttribute("data-rotate", dishName);
-    }
+    dish = JSON.parse(dish);
```

Comment:
consider using a different variable name to distinguish from the input, also, could you move it closer to where it is first used?

Reply:
Good idea